creating content landing page (#2516)
umbraco/UmbracoDocs: Umbraco-StarterKit/Creating-Content/index.md

---

# Creating content

When you are creating content in Umbraco Uno, you will have many tools at your disposal, this page is designed to help you get started on utilizing these tools.

## Getting started

When you want to create content, it is a good idea to think about what tools to use for the job. An example of this is between image widget and gallery widget, image widget only allows you to add one image. However, if you only need one image that is the perfect tool for the job. Gallery widget will allow you to set up a gallery of images. That also means that you can choose to only select one image with that widget, which of these two widgets you choose to use for the job, depends on what you specifically need for this situation.

> The way to best achieve a good content creation flow could be to sit down mark up what you want to create on the page and then choose the tools accordingly.

Author: line 13 ?

When you have chosen what should be on the page and what tools to use, then you can start building from the top to the bottom.

### Manage Media Library

> When creating content, you will most likely need to work with media as well, and to best work, with your media library, this article offers a good piece of insight into how you can do this best[Manage Media Library](Manage-Media-Library/index.md).

Reviewer: ```suggestion
When creating content, you will most likely need to work with media as well. The [Manage Media Library](Manage-Media-Library/index.md) article offers a good piece of insight into how you can do this best.

### Rich Text Editors

> The Rich Text Editor or RTE as people call it is a great way to create content, it is a text editor, that allows you to write longer text pieces with all the awesome formating you would need.

Reviewer: ```suggestion
The Rich Text Editor or RTE as people call it is a great way to create content. It is a text editor, that allows you to write longer text pieces with all the formatting options you would need.
```

> To learn more about what features a  [Rich Text Editor](Rich-Text-Editors/index.md)  offers, it would be a good idea to read this article.

Reviewer: ```suggestion
To learn more about what features the RTE offers, it would be a good idea to read the [Rich Text Editor](Rich-Text-Editors/index.md) article.
```

### Working With Content

> To have the best experience while creating content it is highly recommendable to have a look at this article [Working With Content](Work-With-content/index.md).

Reviewer: ```suggestion
To have the best experience while creating content it is highly recommendable to have a look at the [Working With Content](Work-With-content/index.md) article.
```

### Working With Widgets

> One of the best tools for creating content in Umbraco Uno is the widgets, these allow for a work process that is very straight forward, and they can even be arranged in a grid for an even more slick look.

Reviewer: ```suggestion
When you're creating content in Umbraco Uno you will primarily be working with Widgets. These allow for a work process that is very straight forward, and they can be arranged in a grid for more complexity.
```

> There are a lot of widgets to choose from, a good place to start would be [Working With Widgets](Working-With-Widgets/index.md) and to learn more about the individual [widgets](../Uno-pedia/Widgets/index.md) look in the uno-pedia.

Reviewer: ```suggestion
There are a lot of widgets to choose from. A good place to start would be the [Working With Widgets](Working-With-Widgets/index.md) article.

For a full overview of the Widgets provided with Umbraco Uno see the [Widgets](../Uno-pedia/Widgets/index.md) section in the Uno-pedia.
```
